crow/orders/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db    import models
+
+from users.models import TimeStampedModel
+
+class OrderStatus(models.Model):
+    status = models.CharField(max_length=40)
+
+    class Meta:
+        db_table = "orderstatus"
```

Comment:
지금 컨벤션 하나도 지켜지지 않았습니다. 모든 테이블 클래스명, 테이블명, 변수명 다시 한 번 검토해주세요.
```suggestion
        db_table = "order_status"
```

Reply:
네